bonus/LogToBugsnag.cfc

```diff
@@ -0,0 +1,146 @@
+﻿<!---
+	Adapter Path: taffy.bonus.LogToBugsnag
+	Configuration Options: (structure)
+
+	variables.framework.exceptionLogAdapterConfig = {
+		apiKey = "c9d60ae4c7e70c4b6c4ebd3e8056d2b8",
```

Comment:
Hopefully that's just a sample API key and not one that's actually useful? If it is useful, you should probably trash it and create a new one for your app(s). :)